docs/persistence.md

```diff
@@ -0,0 +1,110 @@
+# Persistence
+
+Generally speaking, adding database storage or persistence to your Probot App will greatly complicate your life. It's perfectly doable, but for most use-cases you'll be able to manage relevant data within GitHub issues, pull requests and your app's configuration file.
+
+For when you absolutely do need external data storage, here are some examples using a few popular database solutions. Your needs will be slightly different from what any example can help with. Below are some code suggestions to help you get started.
```

Comment:
I think maybe ~~`Your needs will be slightly different from what any example can help with. Below are some code suggestions to help you get started.`~~

seems a bit repetitive/unnecessary

Reply:
My intention here is to make it clear that the examples are not the be-all end-all for database connections in a Probot App. I think I just need to shift around some words to make it less repetitive.